Dockerfile

```diff
@@ -1,32 +1,16 @@
-FROM centos:6.7
+FROM mozillabteam/bmo-base:slim
```

Comment:
https://hub.docker.com/r/mozillabteam/bmo-base/ and https://github.com/mozilla-bteam/docker-bmo-base